### ISSUE TITLE
Pin button is on top of Advanced button

### ISSUE THREAD

**Issue by ssspacez** [psifidotos/Latte-Dock#978]
Image: 

![pin](https://user-images.githubusercontent.com/34775187/38600102-5057e95e-3d39-11e8-9d19-c8e69e714373.png)

Ideas:
* Make the pin button smaller
* Move advanced button a few pixels to the left or somewhere else
* Move the pin button to the left side
* Remove pin button

**Comment by psifidotos** [comment]
good catch... i will try to put both in the same ColumnLayout and update the window calculations
